infrastructure/sdk.debug.Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN if [[ ! -z "${USER_UID}" ]]; then \
 RUN /usr/bin/install -d -m 777 /var/run/opcache/debug
 COPY --chown=spryker:spryker infrastructure/debug/php/69-xdebug.ini /usr/local/etc/php/conf.d/69-xdebug.ini
 
+# Set up default stadard for code sniffer
```

Comment:
standard

also, shouldnt this use composer cs-* command instead then?

Reply:
removed